proto/p4/p4runtime.proto

```diff
@@ -114,14 +115,21 @@ message Update {
 
 message Entity {
   oneof entity {
-    TableEntry table_entry = 1;
-    ActionProfileMember action_profile_member = 2;
-    ActionProfileGroup action_profile_group = 3;
-    MeterEntry meter_entry = 4;
-    CounterEntry counter_entry = 5;
+    ExternEntry extern_entry = 1;
+    TableEntry table_entry = 2;
+    ActionProfileMember action_profile_member = 3;
+    ActionProfileGroup action_profile_group = 4;
+    MeterEntry meter_entry = 5;
+    CounterEntry counter_entry = 6;
   }
 }
 
+message ExternEntry {
+  uint32 extern_type_id = 1;
```

Comment:
Agree! That usually happens when this gets deployed and/or there are versioning concerns (old/new client/service, persisted proto dumps, etc. ... standard proto hygiene). I don't think we are there yet, but getting closer.